src/context/PortfolioProvider/PortfolioContext.tsx

```diff
@@ -32,7 +32,7 @@ export const PortfolioProvider = ({ children }: { children: React.ReactNode }) =
     dispatch(portfolio.actions.clear())
     // fetch each account
     accountSpecifiers.forEach(accountSpecifier =>
-      dispatch(portfolioApi.endpoints.getAccount.initiate(accountSpecifier))
+      dispatch(portfolioApi.endpoints.getAccount.initiate(accountSpecifier, { forceRefetch: true }))
```

Comment:
nice catch - i didn't anticipate that we'd be switching wallets quickly and this wouldn't refetch, and would never hit the `onCacheEntryAdded`

Reply:
yeah sure, I will leave a comment here

Reply:
done